src/getty-generator/getty-generator.c

```diff
@@ -30,7 +30,8 @@ static int add_serial_getty(const char *tty) {
         int r;
 
         assert(tty);
-        assert(!path_startswith(tty, "/dev/"));
```

Comment:
This is added before the commit, and now dropped again. Please do not do like that.